src/commissaire/constants.py

```diff
@@ -59,4 +59,6 @@
     'METHOD_NOT_FOUND': -32601,
     'INVALID_PARAMETERS': -32602,
     'INTERNAL_ERROR': -32603,
+    # Custom codes
+    'CONFLICT': -32049,
```

Comment:
The positive space is empty but the spec notes: ```-32000 to -32099	Server error	Reserved for implementation-defined server-errors.```

I'd be OK switching to positive and using ``409`` though.